src/SignalR/clients/csharp/Client.Core/src/HubConnection.cs

```diff
@@ -63,6 +63,31 @@ public partial class HubConnection
         private ConnectionState _connectionState;
         private int _serverProtocolMinorVersion;
 
+        /// <summary>
+        /// Occurs when the connection is closed, either by a call to <see cref="StopAsync(CancellationToken)"/> or a connection error.
```

Comment:
or the server closed the connection gracefully

Reply:
True, will update.